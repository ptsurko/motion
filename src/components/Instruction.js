import React, { PureComponent } from 'react';

class Instruction extends PureComponent {
  render() {
    return (
      <section className="instruction-container">
        <ol className="instruction">
          <li data-step-number="1" className="instruction-step">
            <p>On your mobile device, open your unique URL from the share button below.</p>
          </li>
          <li data-step-number="2" className="instruction-step">
            <p>Click "Start connection".</p>
          </li>
          <li data-step-number="3" className="instruction-step">
            <p>Rotate your device and watch it move in real time! Supports multiple devices.</p>
          </li>
        </ol>
      </section>
    );
  }
};

export default Instruction;
