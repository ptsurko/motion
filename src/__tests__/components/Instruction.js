import React from 'react';
import renderer from 'react-test-renderer';
import Instruction from 'components/Instruction';

it('renders instruction component correctly', () => {
  const tree = renderer.create(
    <Instruction />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
