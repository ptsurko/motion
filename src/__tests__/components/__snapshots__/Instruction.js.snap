// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders instruction component correctly 1`] = `
<section
  className="instruction-container"
>
  <ol
    className="instruction"
  >
    <li
      className="instruction-step"
      data-step-number="1"
    >
      <p>
        On your mobile device, open your unique URL from the share button below.
      </p>
    </li>
    <li
      className="instruction-step"
      data-step-number="2"
    >
      <p>
        Click "Start connection".
      </p>
    </li>
    <li
      className="instruction-step"
      data-step-number="3"
    >
      <p>
        Rotate your device and watch it move in real time! Supports multiple devices.
      </p>
    </li>
  </ol>
</section>
`;
